src/styles/buy.css

```diff
@@ -0,0 +1,22 @@
+.buy__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
+
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.buy__button:hover {
+  border: 1px solid #00acdc;
+  background-color: #fff;
+  color: #00acdc;
+}
```

Comment:
You can move this code inside of `.buy_button` and you can write it like :hover {your style}

Reply:
I don't understand this comment. 
Besides, in the tutorial videos it was taught to style pseudo class (:hover) in this manner, which I did.